tests/test_measures_gold.py

```diff
@@ -0,0 +1,101 @@
+from pandas import read_csv
+
+
+import association_measures.frequencies as fq
+import association_measures.measures as am
+
+
+def test_frequencies():
+
+    df = read_csv("tests/ucs-gold.ds.gz", comment='#', index_col=0,
```

Comment:
I'm not a big fan of binary data in a repository, it'll become larger and larger when this changes. I'd prefer to host this somewhere else and download it, or have a smaller not-binary dataset

Reply:
well the gold-data won't change and I doubt that Stefan will implement any new measures in UCS. but I see your point – I'll take a sample of all rows and check in uncompressed files.